CHANGES/977.bugfix

```diff
@@ -0,0 +1,4 @@
+Introduced a repair command for fixing the media types of manifests that could be incorrectly
+determined during the sync procedure when running the following Y releases of pulp_container:
+2.13.1, 2.10.5, 2.9.6. Run the command ``pulpcore-manager container-repair-media-type`` to
```

Comment:
i would omit the versions, this issue is there for quite some time.

Reply:
Agree, also i would expect a version range on every branch. Better to just skip it.